examples/basics/visuals/polygon_prototype.py

```diff
@@ -0,0 +1,49 @@
+from vispy import app
```

Comment:
Actually, this is a scenegraph example (all of the existing examples are visual examples) so just move it to `examples/basics/scene`. Please make sure the existing visual examples work as well, though.